webapp/mixins/scrollToAnchor.js

```diff
@@ -0,0 +1,20 @@
+import { scrollTo } from 'vue-scrollto'
+
+export default {
+  watch: {
+    $route(to, from) {
+      const anchor = to && to.hash
+      if (!this.checkAnchor(anchor)) return
+      setTimeout(() => {
+        scrollTo(anchor, 1000)
```

Comment:
@roschaefer IMO it's a good idea to add `vue-scrollto` as a module to 'nuxt.config.js' as described [here](https://github.com/rigor789/vue-scrollto#nuxtjs).

I'd pass default options:
```
{
  duration: 1000,
  offset: -100, // to compensate fixed navbar height
}
```

Reply:
From my limited experience working with `nuxt.scrollBehavior`, what I observed was that it is not called if you are on the same page, only on page transitions, so maybe the weird jump is not related to that??
https://router.vuejs.org/guide/advanced/scroll-behavior.html#scroll-behavior